src/components/accessories/patientOperation/operationForm/OperationRowForm.tsx

```diff
@@ -74,7 +82,7 @@ const OperationRowForm: FC<OperationRowProps> = ({
   const dateFieldHandleOnChange = useCallback(
     (fieldName: string) => (value: any) => {
       setFieldValue(fieldName, value);
-      formik.validateField("opDate");
+      formik.setFieldTouched(fieldName);
     },
     [setFieldValue, initialValues.opDate]
   );
```

Comment:
I think that should be better to have a second callback which takes care of the onBlur method.